jscomp/core/lam_compile_external_call.ml

```diff
@@ -30,6 +30,10 @@ let splice_fn_apply fn args =
   E.runtime_call Js_runtime_modules.caml_splice_call "spliceApply"
     [ fn; E.array Immutable args ]
 
+let splice_fn_new_apply fn args =
```

Comment:
How about the name `splice_new_apply`?

Reply:
And why is it called `splice` and not `variadic`. Or are the two concepts not referring to the same mechanism?

Reply:
I used `splice_fn_new_apply` because of the existing `splice_obj_fn_apply`. Let's rename it too.

and it's because `[@variadic]` is used to be called as `[@bs.splice]`, and it seems the code has not changed since then (yes, splice = variadic).